drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -133,6 +132,28 @@ class DragAndDropBlock(
         enforce_type=True,
     )
 
+    showanswer = String(
+        display_name=_("Show Answer"),
+        help=_("Defines when to show the answer to the problem. "
+               "A default value can be set in Advanced Settings."),
+        scope=Scope.settings,
+        default=SHOWANSWER.FINISHED,
+        values=[
```

Comment:
Should I keep "showanswer" values list (ie, always, finish, etc)?
If the answer is yes,  How do I  add translations for display_name?